### PR TITLE
feat: add after hooks

### LINKED ISSUE #182: Add `AfterFixedPriceAuctionCreated ` and `AfterBatchAuctionCreated` hooks
## Summary

Adding `AfterFixedPriceAuctionCreated` and `AfterBatchAuctionCreated` hooks similar to currently implemented `BeforeFixedPriceAuctionCreated ` and `BeforeBatchAuctionCreated` hooks but with `auctionID` value available.

## Problem Definition

Current `BeforeFixedPriceAuctionCreated ` and `BeforeBatchAuctionCreated` hooks don't give access to the `auctionID` since the auction is not yet created at the time the hook is called.
For some specific purpose, the `auctionID` must be accessed.
Example: in `spn` https://github.com/tendermint/spn we wish to access the `auctionID` to associate an auction ID with a related campaign ID when an auction is created.

## Proposal

Add `AfterFixedPriceAuctionCreated ` and `AfterBatchAuctionCreated` hooks similar to `BeforeFixedPriceAuctionCreated ` and `BeforeBatchAuctionCreated`  but that includes the auction ID

## Tasks

- 

## References

-
____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

### PR DESCRIPTION
**Pull request by jaybxyz**
## Description

closes: #182

## Tasks

- [x] Add `AfterFixedPriceAuctionCreated` and `AfterBatchAuctionCreated` hooks
- [x] Update docs 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by jaybxyz** [comment]
@jeonghoyeo7 Can you review this when you have time?

**Review by lumtis — APPROVED** [review]
Thanks for having looked into this so quickly🙏

File: x/fundraising/keeper/auction.go [hunk 1]
```diff
@@ -172,6 +185,22 @@ func (k Keeper) CreateBatchAuction(ctx sdk.Context, msg *types.MsgCreateBatchAuc
...
+		auction.StartPrice,
+		auction.MinBidPrice,
+		auction.SellingCoin,
+		auction.PayingCoinDenom,
+		auction.VestingSchedules,
+		auction.MaxExtendedRound,
+		auction.ExtendedRoundRate,
+		auction.StartTime,
+		auction.EndTimes[0],
+	)
```
Comment (jeonghoyeo7) [inline x/fundraising/keeper/auction.go:202]: Isn't `MatchedPrice` needed? Even though it seems not necessary when created, `NewBatchAuction` has the following.
```
func NewBatchAuction(baseAuction *BaseAuction, minBidPrice sdk.Dec, matchedPrice sdk.Dec, maxExtendedRound uint32, extendedRoundRate sdk.Dec) *BatchAuction {
	return &BatchAuction{
		BaseAuction:       baseAuction,
		MinBidPrice:       minBidPrice,
		MatchedPrice:      matchedPrice,
		MaxExtendedRound:  maxExtendedRound,
		ExtendedRoundRate: extendedRoundRate,
	}
}
```

**Review by jeonghoyeo7 — COMMENTED** [review]
Thank you, JayB! I just left one minor comment.

File: x/fundraising/keeper/auction.go (hunk 1 above)
Comment (jaybxyz) [reply x/fundraising/keeper/auction.go:202]: Yes, it is used when a batch auction ends. `MatchedPrice` is not required in message level meaning that an auctioneer doesn't need to input the value to create a batch auction.

**Pull request merged**